src/apps/inventory/models.py

```diff
@@ -14,6 +16,12 @@ class Product(models.Model):
     description = models.TextField(blank=True)
     category = models.ForeignKey(Category, on_delete=models.PROTECT)
     price = models.DecimalField(max_digits=10, decimal_places=2)
+    seller = models.ForeignKey(
```

Comment:
Maybe lets call it "created_by". Please also add information about when product was created.

Reply:
to be honest i like seller more, created_by is a little misleading maybe "in the future" we will add this field for example for the country where the product was created or the brand of manufacturer

Reply:
Time added